documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -23,7 +23,13 @@ This document defines guidelines for Operator API Exposure Platforms to manage C
 
 Some CAMARA APIs process Personal Data and according to local regulations may require a “legal basis” to do so (e.g. "legitimate interest”, “contract”, “consent”). Operator API Exposure Platforms exposing CAMARA APIs should be built with a privacy-by-design approach to fully comply with any relevant data protection requirements and regulations, such as [GDPR](https://gdpr-info.eu/) in Europe.
 
-**CAMARA API access will be secured using [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html) (OIDC) on top of [OAuth 2.0 protocol](https://datatracker.ietf.org/doc/html/rfc6749) following the [CAMARA Security and Interoperability Profile](CAMARA-Security-Interoperability.md)**.
+CAMARA API access will be secured by the protocols described in the [CAMARA Security and Interoperability Profile](CAMARA-Security-Interoperability.md)
+- [OpenID Connect Authorization Code Flow](CAMARA-Security-Interoperability.md#oidc-authorization-code-flow) (OIDC) adds User authentication and consent collection on top of the [OAuth 2.0 protocol](https://datatracker.ietf.org/doc/html/rfc6749) in typical use cases where end-user's device sends an Authentication request to the Authorization Server starting User authentication and consent collection.
```

Comment:
```suggestion
- [OpenID Connect Authorization Code Flow](CAMARA-Security-Interoperability.md#oidc-authorization-code-flow) (OIDC) adds User authentication and Consent management on top of the [OAuth 2.0 protocol](https://datatracker.ietf.org/doc/html/rfc6749) in typical use cases where End-User's Device sends an Authentication request to the Authorization Server starting User authentication and Consent collection.
```

Reply:
I changed end-user to End-User and consent to Consent because they are a glossary terms

Reply:
Thanks for the review. The PR now just lists the protocols. Please review again.